### PR TITLE
[Java] Make the default ingress channel null. Add constants to be used in cluster tests.

### PR DESCRIPTION
**Pull request by mikeb01**

### CODE REVIEW

File: aeron-system-tests/src/test/java/io/aeron/cluster/MultiModuleSharedDriverTest.java [hunk 1]
```diff
@@ -126,6 +127,8 @@ public void shouldSupportTwoSingleNodeClusters()
                 final EgressListener egressListener = (clusterSessionId, timestamp, buffer, offset, length, header) ->
                     egress.set(buffer.getStringWithoutLengthAscii(offset, length));
 
+                Tests.sleep(1_000);
```
Comment (mjpt777) [inline aeron-system-tests/src/test/java/io/aeron/cluster/MultiModuleSharedDriverTest.java:130]: Why is this needed?

Comment (mikeb01) [reply aeron-system-tests/src/test/java/io/aeron/cluster/MultiModuleSharedDriverTest.java:130]: It's not, that was me trying to debug a failure.  Removed.
